src/pages/public_registration/action_accordions/SupportAccordion.tsx

```diff
@@ -45,11 +63,26 @@ export const SupportAccordion = ({ registration, supportTicket, addMessage, refe
       elevation={3}>
       <AccordionSummary sx={{ fontWeight: 700 }} expandIcon={<ExpandMoreIcon fontSize="large" />}>
         {t('my_page.messages.types.GeneralSupportCase')}
+        {supportTicket && ` - ${t(`my_page.messages.ticket_types.${supportTicket.status}`)}`}
       </AccordionSummary>
       <AccordionDetails sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
         {supportTicket && (
           <>
             <TicketAssignee ticket={supportTicket} refetchTickets={refetchData} />
+            {userIsCurator && window.location.pathname.startsWith(UrlPathTemplate.Tasks) && (
+              <LoadingButton
+                sx={{
+                  alignSelf: 'end',
+                  width: 'fit-content',
+                  bgcolor: 'white',
+                  display: supportTicket.status === 'Completed' ? 'none' : 'visible',
```

Comment:
hvorfor skjule den med CSS i stedet for i legge til sjekken utenfor knappen slik at den ikke rendres i heletatt?

Reply:
Whops